samples/ClassroomLabs/Modules/Library/Tests/RunPesterTests.ps1

```diff
@@ -64,12 +66,5 @@ else {
         $jobs += Start-Job -Script $invokePesterScriptBlock -ArgumentList $_, $PSScriptRoot, $VerboseTests
     }
 
-    $jobs | ForEach-Object {
-        Write-Output "-----------------------------------------------------------------"
-        $result = Receive-Job -Job $_ -Wait -Verbose
-
-        if ($result.FailedCount -ne 0) {
-            Write-Error "Pester returned errors for $($result.TestResult.Describe) - $($result.TestResult.Context)"
```

Comment:
I'll check out the latest results - last time I tried this approach, even if there were test failures they weren't reported (the pipeline always showed as passed even if there were failures... )  When a test fails, does it show the pipeline as failed correctly?

Reply:
As discussed, it works by failing the 'publish test result' step in the pipeline.